includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -48,9 +48,22 @@ public function __construct( WC_Payments_Account $account, $gateway ) {
 	 */
 	public function init(): string {
 		global $product;
-		$price           = $product->get_price();
-		$currency_code   = get_woocommerce_currency();
-		$billing_country = WC()->countries->get_base_country();
+		$price             = $product->get_price();
+		$currency_code     = get_woocommerce_currency();
+		$billing_country   = WC()->countries->get_base_country();
+		$all_variation_ids = $product->get_children();
+
+		$variations_price_list = array_reduce(
```

Comment:
Regular foreach loop will be a better fit here.

Reply:
Replaced. Thanks for pointing this out. 0129a74